Artifacts/windows-firewall/Artifactfile.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+  "title": "Execute basic Windows Firewall operations",
+  "description": "Artifact to execute basic Windows firewall commands. This could be useful if you mistakenly enable rules which would deny access to your VM",
+  "publisher": "Milan Musec",
+  "tags": [
+      "Windows"
+  ],
+  "targetOsType": "Windows",
+  "parameters": {
+    "firewallOperation": {
+      "type": "string",
+      "displayName": "Firewall Operation:",
+      "description": "Select firewall operation",
+      "defaultValue": "-- none --",
```

Comment:
Consider making the `defaultValue` a blank and remove `"-- none --"` from the `allowedValues` list and don't add a blank as an allowed value. This way the parameter becomes mandatory and the artifact is not allowed to be executed for no reason, to simply get a message like `"No operation executed"`.

Reply:
done